Sources/PointFree/Account/Account.swift

```diff
@@ -91,22 +128,35 @@ private let creditsView = View<(Stripe.Subscription?, Database.User, [Database.E
             h2([`class`([Class.pf.type.title4])], ["Episode Credits"]),
             p([
               "Episode credits allow you to see subscriber-only episodes before commiting to a full ",
-              text("subscription. You currently have \(pluralizedCredits(count: currentUser.episodeCreditCount)) "),
+              text("subscription. You currently have \(pluralizedCredits(count: data.currentUser.episodeCreditCount)) "),
               "remaining."
-              ]),
-            p([
-              "To get all past and future episodes, ",
-              a([`class`([Class.pf.colors.link.purple]), href(path(to: .pricing(nil, expand: nil)))], ["become"]),
-              " a subscriber today!"
```

Comment:
we now hide this blurb for owner's of subscriptions, so moved that logic into `subscribeCallout`